modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcp-bigquery-connector.adoc

```diff
@@ -80,59 +97,73 @@ _optional_ advanced connector configuration properties by selecting *Show advanc
 on the *Create Connector* page:
 
 |===
-| Property | Description
+| Property name | Property key | Description
 
 | `Auto create tables`
+| `autoCreateTables`
 | Automatically create BigQuery tables if they don't already exist. If the table does not exist, then it is created based on the record schema.
 
 | `Topic to table map`
+| `topic2TableMap`
 | Map of topics to tables. Format: comma-separated tuples, for example `topic1:table1,topic2:table2`.
 
 | `Allow new BigQuery fields`
+| `allowNewBigQueryFields`
 | If true, new fields can be added to BigQuery tables during subsequent schema updates.
 
 | `Allow BigQuery required field relaxation`
+| `allowBigQueryRequiredFieldRelaxation`
 | If true, fields in the BigQuery schema can be changed from `REQUIRED` to `NULLABLE`.
 
 | `Upsert enabled`
+| `upsertEnabled`
 | Enables upsert functionality on the connector.
 
 | `Delete enabled`
+| `deleteEnabled`
 | Enable delete functionality on the connector.
 
 | `Kafka key field name`
+| `kafkaKeyFieldName`
 | The name of the BigQuery table field for the Kafka key. Must be set when upsert or delete is enabled.
 
 | `Time partitioning type`
+| `timePartitioningType`
 | The time partitioning type to use when creating tables.
 
 | `BigQuery retry attempts`
+| `bigQueryRetry`
 | The number of retry attempts made for each BigQuery request that fails with a backend or quota exceeded error.
 
 | `BigQuery retry attempts interval`
+| `bigQueryRetryWait`
 | The minimum amount of time, in milliseconds, to wait between BigQuery backend or quota exceeded error retry attempts.
 
 | `Error tolerance`
+| `errors.tolerance`
 | Error tolerance response during connector operation. Default value is `none` and signals that any error will result in an immediate connector task failure. Value of `all` changes the behavior to skip over problematic records.
 
 | `Dead letter queue topic name`
+| `errors.deadletterqueue.topic.name`
 | The name of the topic to be used as the dead letter queue (DLQ) for messages that result in an error when processed by this sink connector, its transformations, or converters. The topic name is blank by default, which means that no messages are recorded in the DLQ.
 
 | `Dead letter queue topic replication factor`
+| `errors.deadletterqueue.topic .replication.factor`
```

Comment:
I added space to some properties with long names only to wrap them in the table.